toolkit/scripts/sodiff/mariner-sodiff.sh

```diff
@@ -58,7 +83,9 @@ done
 # Obtain a list of unique packages to be updated
 2>/dev/null cat "$sodiff_out_dir"/require* | sort -u > "$sodiff_out_dir"/sodiff-intermediate-summary.txt
 
-rm "$sodiff_out_dir"/require*
+if [ ls "$sodiff_out_dir"/require* 1> /dev/null 2>&1 ]; then
+    rm "$sodiff_out_dir"/require*
+fi
```

Comment:
What is the purpose of this check? If you're trying to ignore missing files without an error, then `rm` has the `-f` flag.
```suggestion
rm -f "$sodiff_out_dir"/require*
```

Reply:
changed to use -f